Kernel/Bus/PCI/GenericInitializer.cpp

```diff
@@ -241,11 +240,25 @@ void initialize()
 
                 u32 interrupt_controller_phandle = MUST(map_stream.read_cell());
                 auto const* interrupt_controller = device_tree.phandle(interrupt_controller_phandle);
+#if ARCH(AARCH64)
+                // The interrupt controller handle contains two extra cells on aarch64, but they don't
+                // contain anything of interest.
+                MUST(map_stream.discard(sizeof(u32) * 2));
+#endif
                 VERIFY(interrupt_controller);
 
                 auto interrupt_cells = interrupt_controller->get_property("#interrupt-cells"sv)->as<u32>();
+#if ARCH(RISCV64)
                 VERIFY(interrupt_cells == 1 || interrupt_cells == 2);
                 u64 interrupt = MUST(map_stream.read_cells(interrupt_cells));
+#elif ARCH(AARCH64)
+                VERIFY(interrupt_cells == 3);
+                MUST(map_stream.discard(sizeof(u32))); // This is the IRQ type.
+                u64 interrupt = MUST(map_stream.read_cell());
+                MUST(map_stream.discard(sizeof(u32))); // This is the trigger type.
+#else
+#    error "Unknown architecture"
```

Comment:
nit: I know we do this in other places, but the quotation marks are unnecessary.